docs/hu/docs/index.md

```diff
@@ -0,0 +1,470 @@
+
+* Szerkesztő támogatást, beleértve:
+    * Szövegkiegészítés.
+    * Típus ellenőrzés.
+* Adatok validációja:
+    * Automatikus és érthető hibák amikor az adatok hibásak.
+    * Validáció mélyen ágyazott objektumok esetén is.
+* Bemeneti adatok<abbr title="also known as: serialization, parsing, marshalling"> átváltása</abbr> : a hálózatról érkező Python adatokká és típusokká. Adatok olvasása következő forrásokból:
+    * JSON.
+    * Cím paraméterek.
+    * Query paraméterek.
+    * Cookie-k.
+    * Header-ök.
+    * Formok.
+    * Fájlok.
+* Kimeneti adatok <abbr title=" más néven: serialization, parsing, marshalling">átváltása</abbr>: Python adatok is típusokról hálózati adatokká:
+    * válts át Python típusokat (`str`, `int`, `float`, `bool`, `list`, etc).
+    * `datetime` csak objektumokat.
+    * `UUID` objektumokat.
+    * Adatbázis modelleket.
+    * ...És sok mást.
+* Automatikus interaktív dokumentáció, beleértve két alternatív dokumentációt is:
+    * Swagger UI.
+    * ReDoc.
+
+---
+
+Visszatérve az előző kód példához. A **FastAPI**:
+
+* Validálja hogy van egy `item_id` mező a `GET` és `PUT` kérésekben.
+* Vali Dálya hogy az `item_id` `int` típusú a `GET` és `PUT` kérésekben.
+    * Ha nem akkor látni fogunk egy tiszta hibát ezzel kapcsolatban.
+* ellenőrzi hogyha van egy opcionális query paraméter `q` névvel (azaz `http://127.0.0.1:8000/items/foo?q=somequery`) `GET` kérések esetén.
+    * Mivel a `q` paraméter `= None`-al van deklarálva, ezért opcionális.
+    * `None` nélkül ez a mező kötelező lenne (mint például a body `PUT` kérések esetén).
+* a `/items/{item_id}` címre érkező `PUT` kérések esetén, a JSON-t a következőképpen olvassa be:
+    * Ellenőrzi hogy létezik a kötelező `name` nevű attribútum és `string`.
+    * Ellenőrzi hogy létezik a kötelező `price` nevű attribútum és `float`.
+    * Ellenőrzi hogy létezik a `is_offer` nevű opcionális paraméter, ami ha létezik akkor `bool`
+    * Ez ágyazott JSON objektumokkal is működik
+* JSONről való automatikus konvertálás.
+* dokumentáljuk mindent OpenAPI-al amit használható:
+    * Interaktív dokumentációs rendszerekkel.
+    * Automatikus kliens kód generáló a rendszerekkel, több nyelven.
+* Hozzá tartozik kettő interaktív dokumentációs web felület.
+
+---
+
+We just scratched the surface, but you already get the idea of how it all works.
+
+Próbáld kicserélni a következő sorban:
+
+```Python
+    return {"item_name": item.name, "item_id": item_id}
+```
+
+...ezt:
+
+```Python
+        ... "item_name": item.name ...
+```
+
+...err:
+
+```Python
+        ... "item_price": item.price ...
+```
+
+... És figyeld meg hogy a szerkesztőt automatikusan tudni fogja a típusokat és kiegészíti azokat:
+
+![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
+
+Teljesebb példákért és funkciókért tekintsd meg a<a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - User Guide</a>.
+
+**Spoiler veszélz**: a tutorial - user guidehoz tartozik:
```

Comment:
```suggestion
**Spoiler veszély**: a Tutorial - User Guidehoz tartozik:
```
maybe we could use the "izelito" word here?
something like 
"**Spoiler veszély**: izelito a Tutorial - User Guide -bol:"?